Assets/Scripts/Game Scene/Obstacle/Obstacle.cs

```diff
@@ -27,6 +27,11 @@ public void InitiateMovement(int difficultyChosen, Action<Obstacle> _releaseCall
         isActive = true;
     }
 
+    public void StopMovement()
+    {
+        isActive = false;
+    }
+
     private void Update()
```

Comment:
You could use a Coroutine here. The script will call for an update even when the obstacle is not been used but is active.

Reply:
I tried that, but for some reason the obstacles weren't moving 😪 after the project is finished, I'll comeback here to look at this with more detail.